handler/pullRequestHandler.go

```diff
@@ -143,3 +196,7 @@ func hasUnsigned(req types.PullRequestOuter, client *github.Client) (bool, error
 func isSigned(msg string) bool {
 	return strings.Contains(msg, "Signed-off-by:")
 }
+
+func missingDescription(pr types.PullRequest) bool {
```

Comment:
Can you call this `hasDescription` please?

Reply:
updated. Changing the name redefines this function, so I've updated it to return `> 0` rather than the previous implementation of `== 0`

I've updated the calling function and the tests to reflect this new change